docs/pywbemcligeneraloptions.rst

```diff
@@ -38,25 +36,25 @@ information on each command-group arguments and options):
   A new server (``myserver``)may be defined and saved in the connection file with a
```

Comment:
Insert space between "(``myserver``)" and "may".

Reply:
DONE